packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts

```diff
@@ -33,6 +33,12 @@ export abstract class CollectorExporterBrowserBase<
   ExportItem,
   ServiceRequest
 > {
+  private DEFAULT_HEADERS = {
```

Comment:
why do you think it's better to keep defaults on instance instead in utils  ?

Reply:
correct me if i am wrong.

Who owns the config ? 
- The `CollectorExporterBrowserBase`
- Owning the data means working with the data and correcting the data if needed, like applying default headers or type for blob property bag.

What are the methods `sendWithXhr`, `sendWithBeacon` under 
opentelemetry-js/packages/opentelemetry-exporter-collector/src/platform/browser/util.ts ? 

- Are roles with the responsibility to send data with a protocol. 
- Protocol is the name of each function and data is the arguments that each function define.